other/materials_designer/create_interface_with_no_strain_matching.ipynb

```diff
@@ -43,7 +43,26 @@
     "# Enable interactive selection of terminations via UI prompt\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.    TERMINATION_PAIR_INDEX = 0
</pre><p>TERMINATION_PAIR_INDEX # Will be overriden in interactive selection is used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>ok</p>